index.html

```diff
@@ -4,15 +4,264 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet"/>
   <link rel="stylesheet" href="styles/main.css" />
 
   <title>JT's Capstone One (Module one)</title>
 </head>
 <body>
-  <h1>Jud&aacute; Teixeira's Capstone One</h1>
-  <p>
-    Hi there
-  </p>
+  <div class="grid-container">
+    <header>  <!-- START OF HEADER -->
```

Comment:
- [ ] Let's take around to your readme file,  I think you can still add a few more things; your project description is not customized to the project at all, please give a short summary of what the website is for.
- [ ] I would suggest you change the project title to the actual project topic, it can be something like `EDM marketing conference` instead of JT's..... what is there 😄 . please change it.
- 
<img width="880" alt="desr" src="https://user-images.githubusercontent.com/79058364/157601051-9e64b68c-5d22-46c1-ac8a-4e018482c2b3.png">

- [OPTIONAL ] I can not see your readme file on this branch, I think it is because you pushed it to the main branch before you create a feature branch, as advice, this is not a best practice at all, please take care and keep the main branch clean up to to the last and final code you should merge. 

- [ ] Please add an acknowledgment to the owner of the design, right down at the bottom of the readme, we could see it there, you can refer to the microverse readme template and customize it to your project.